README.md

```diff
@@ -23,26 +23,55 @@ in an experimental and research phase and cannot currently be used traditionally
 source code base for general changes.
 ```
 
-**This repo does not include any assets or code necessary for compiling the ROM. A prior copy of the game is required to extract the required assets.**
+This is a WIP **decompilation** of ***The Legend of Zelda: Majora's Mask***. The purpose of the project is to recreate a source code base for the game from scratch, using information found inside the game along with static and/or dynamic analysis. **It is not producing a PC port.** For more information you can get in touch with the team on our [Discord server](https://discord.zelda64.dev).
 
-This is a decompilation of Legend of Zelda: Majora's Mask (US) 1.0
+The only build currently supported is N64 US, but other versions are planned to be supported.
 
-It builds the following ROM:
+It currently builds the following ROM:
 * mm.us.rev1.rom.z64 `md5: 2a0a8acb61538235bc1094d297fb6556`
 
+**This repo does not include any assets or assembly code necessary for compiling the ROM. A prior copy of the game is required to extract the required assets.**
+
 Please refer to the following for more information:
 
 - [Website](https://zelda64.dev/)
 - [Discord](https://discord.zelda64.dev/)
 - [How to Contribute](CONTRIBUTING.md)
 
+
 ## Installation
 
 ### Windows
 
 For Windows 10, install WSL and a distribution by following this
 [Windows Subsystem for Linux Installation Guide](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
-We recommend using Debian or Ubuntu 18.04 Linux distributions.
+We recommend using Debian or Ubuntu 20.04 Linux distributions.
+
+
+### MacOS
+
+For macOS, use Homebrew to install the following dependencies:
+
+* coreutils
+* make
+* python3
+* libpng
+
```

Comment:
```suggestion
* libpng
* clang-format@11
* bash
```
From working on my mac I have found that it would be helpful to add these here since they will also be necessary for submitting PRs